src/lightning/fabric/utilities/imports.py

```diff
@@ -25,6 +25,6 @@
 # 2. The inspection mode via `python -i`: https://stackoverflow.com/a/6879085/1162383
 _IS_INTERACTIVE = hasattr(sys, "ps1") or bool(sys.flags.interactive)
 
-_TORCH_GREATER_EQUAL_1_12 = compare_version("torch", operator.ge, "1.12.0")
-_TORCH_GREATER_EQUAL_1_13 = compare_version("torch", operator.ge, "1.13.0")
+_TORCH_GREATER_EQUAL_1_12 = compare_version("torch", operator.ge, "1.12.0", use_base_version=True)
+_TORCH_GREATER_EQUAL_1_13 = compare_version("torch", operator.ge, "1.13.0", use_base_version=True)
```

Comment:
This change allows a different bug:

If we use an api added in 1.13.0+b
But the user has 1.13.0+a
Where a is an earlier version to b
There will be an error

Reply:
So I suggest that we don't do this and we just recommend upgrading instead. Meaning we don't support old nightly or pre-release versions

Reply:
The user in the linked issue is using a standard docker image from nvidia: nvcr.io/nvidia/pytorch:22.10-py3
This means we won't support any of these?

Reply:
I guess. I wonder why they use these PyTorch installations. One improvement we could do would be to warn the user about this.

Reply:
I am using these images for the sake of repeatability in my experiments and because all the packages are 
 working out of the box (no need to manage conda/pip requirements, just run `docker run nvcr.io/nvidia/pytorch:22.10-py3 python my_script.py`).

Reply:
@carmocca As an alternative, should we replace the conditional check with a try-except? It's not a real fix but it would at least mitigate the issue for LM instantiation.

Reply:
I don't think that's a good idea, it wouldn't work anywhere else where we have this problem.

Could we detect if we are using one of these special images, and then use the base version?. I'd like to understand why these specific commits are installed. I looked and they don't seem to be the last commit of their respective releases.

Reply:
As this is related to officila Nvidia/PyTorch images, I would roll this change with `use_base_version`

Reply:
I added a comment above the changed lines with an explanation for the issue with a reference to this PR

Reply:
@awaelchli This introduced a failing workflow in master (build-NGC) exactly because of this issue. The NGC 1.13 image installs an 1.13 commit that doesn't include a feature included in the true 1.13 release:

https://github.com/Lightning-AI/lightning/actions/runs/4358626506/jobs/7619447910#step:3:1685

Reply:
Sure. This method will go away in future releases anyways